openjdk/playlist.xml

```diff
@@ -90,6 +90,99 @@
 			<impl>hotspot</impl>
 		</impls>
 	</test>
+	<test>
+		<testCaseName>hotspot_container</testCaseName>
+		<variations>
+			<variation>Mode150</variation>
+			<variation>Mode650</variation>
+			<variation>Mode1000</variation>
+		</variations>
```

Comment:
This currently is set to running [Mode1000](https://github.com/adoptium/TKG/blob/master/resources/modes.xml#L4617-L4625) NoOptions for 32bit platforms and [Mode150](https://github.com/adoptium/TKG/blob/master/resources/modes.xml#L954-L962) `-XX:+UseCompressedOops` / [Mode650](https://github.com/adoptium/TKG/blob/master/resources/modes.xml#L4279-L4287) `-XX:-UseCompressedOops` for 64 bit platforms (so will be run twice on 64bit platforms).  If this is not a useful set of variations for these targets, this could be changed to 
```
<variations>
        <variation>NoOptions</variation>
</variations>
```
To run once, with no additional JVM options added.

Reply:
OK, thanks! Those variations don't seem useful. I'll update.